test/ui-testing/interfaces.js

```diff
@@ -92,8 +92,9 @@ module.exports.test = (uiTestCtx) => {
           .wait('[data-test-find-interfaces-modal]')
           .wait('#input-interface-search')
           .type('#input-interface-search', interfaceName)
-          .click('#clickable-search-agreements')
-          .waitUntilNetworkIdle(1000)
+          .wait('[data-test-search-and-sort-submit]')
```

Comment:
This doesn't exist as of the SASQ refactor but `#clickable-search-agreements` (what it's being changed _from_) does. 

Any chance you updated these tests _before_ the SASQ refactor? Can you rerun these to confirm they still pass?